lib/ansible/modules/network/nxos/nxos_hsrp.py

```diff
@@ -143,10 +143,14 @@ def execute_show_command(command, module):
 
 def apply_key_map(key_map, table):
     new_dict = {}
-    for key in table:
+    for key, value in table.items():
```

Comment:
So this is an astonishingly small nit, but we still don't need to get `value` from table, `for key in table:` will work fine here.

But that's probably not anything worth blocking the PR on